srcpkgs/e2guardian/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'e2guardian'
+pkgname=e2guardian
+version=5.3.2
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --enable-clamd=yes --with-proxyuser=_e2guardian --with-proxygroup=_e2guardian --sysconfdir=/etc --localstatedir=/var --enable-icap=yes --enable-commandline=yes --enable-email=yes --enable-ntlm=yes --mandir=${prefix}/share/man --infodir=${prefix}/share/info --enable-pcre=yes --enable-sslmitm=yes"
+conf_files="/etc/e2guardian/*.conf
+/etc/e2guardian/site.story
+/etc/e2guardian/preauth.story
+/etc/e2guardian/site.story
+/etc/e2guardian/lists/authplugins/*
+/etc/e2guardian/lists/bannedrooms/*
+/etc/e2guardian/lists/contentscanners/*
+/etc/e2guardian/lists/phraselists/*/*
+/etc/e2guardian/authplugins/*
+/etc/e2guardian/contentscanners/*
+/etc/e2guardian/downloadmanagers/default.conf"
+make_dirs="/var/log/e2guardian 0755 _e2guardian _e2guardian"
+hostmakedepends="automake pkg-config"
+makedepends="zlib-devel pcre-devel libressl-devel"
+short_desc="E2guardian is a web content filter proxy with multiple server modes"
```

Comment:
Fix the xlint errors.

Reply:
I ran xlint on the template, but it didn't throw an error for me. Could you please point out what is wrong with that part?

Reply:
Please remove the package name from description, just use `Web content filter proxy with multiple server modes`.